test/dash.js

```diff
@@ -4,8 +4,11 @@ var parse = require('../');
 var test = require('tape');
 
 test('-', function (t) {
```

Comment:
Just noticed the new `---` value tests don't belong in the existing test group, which is concisely titled `-`. I'll move them.

Reply:
Refactored so logically grouped.